ocean_lib/web3_internal/test/test_contract_handler_addresses.py

```diff
@@ -3,13 +3,12 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# This file does tests for method "get_contracts_addresses()", that's all.
```

Comment:
Can you please replace this comment with a docstring?

Reply:
fixed